Graphs/KannsAlgorithm.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @author {RaviSadam}
+ * @name kannsAlgorithm
+ * @description -
+ * Kann's Algorithm implementation in JavaScript
+ * @summary
+ * Kann's Algorithm is used for topological sorting in directed acyclic graphs
+ *
+ * @param graph - Graph [[v1,v2],[v3,v4,v5]..]
+ * @param n - number of vertices
+ * @returns {Array} - Empty array if cycle is detected or else result array;
+ *
+ */
+
+export function kannsAlgorithm(graph, n) {
+  if (n === null || n === undefined) throw Error('Invalid n was given')
+  const inorder = Array(n).fill(0)
```

Comment:
Misleading name, a better name would be `indegree` or `indeg` for short.

Reply:
Updated to inDegree